pkg/ruler/memstore_test.go

```diff
@@ -2,11 +2,12 @@ package ruler
 
 import (
 	"context"
-	"github.com/prometheus/common/model"
-	"github.com/prometheus/prometheus/model/rulefmt"
 	"testing"
 	"time"
 
+	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/model/rulefmt"
```

Comment:
I believe it must be grouped with the group below

Reply:
I'm just sticking with what the linter says